kong/plugins/cors/schema.lua

```diff
@@ -7,6 +7,7 @@ return {
     methods = { type = "array", enum = { "HEAD", "GET", "POST", "PUT", "PATCH", "DELETE" } },
     max_age = { type = "number" },
     credentials = { type = "boolean", default = false },
-    preflight_continue = { type = "boolean", default = false }
+    preflight_continue = { type = "boolean", default = false },
+    origin_domains = { type = "array" }
```

Comment:
additionally, not knowing CORS in-depth, shouldn't `origin` and `origin-domains` be mutually exclusive?